pkgs/applications/misc/openrgb/default.nix

```diff
@@ -26,6 +26,10 @@ mkDerivation rec {
     mkdir -p $out/etc/udev/rules.d
     cp 60-openrgb.rules $out/etc/udev/rules.d
 
+    mkdir -p "$out/share"/{applications,icons/hicolor/128x128/apps}
+    install qt/OpenRGB.desktop "$out/share/applications/"
+    install qt/OpenRGB.png "$out/share/icons/hicolor/128x128/apps"
+
```

Comment:
You can drop the `mkdir` bit and have `install` create the directories:
```
install -Dm444 -t $out/share/applications qt/OpenRGB.desktop
install -Dm444 -t $out/share/icons/hicolor/128x128/apps qt/OpenRGB.png
```

The `Exec` part of the .desktop file is pointing to the proper path in the nix store?

Reply:
This is quite neat, thanks! I got rid of the `mkdir` part in favor of the nicer `install` syntax.

Regarding the `Exec` part, it simply runs `openrgb` from the environment path instead of having an absolute path. From what I understand, this is useful for users building their own derivations as it does not reference a specific package in the store. But feel free to correct me if this is not the case.

Reply:
there's no path, but I think that's desired. The correct command name is there.